lte/gateway/c/session_manager/test/test_polling_pipelined.cpp

```diff
@@ -0,0 +1,176 @@
+/**
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include <folly/io/async/EventBaseManager.h>
+#include <gtest/gtest.h>
+#include <lte/protos/session_manager.grpc.pb.h>
+#include <string.h>
+#include <time.h>
+
+#include <memory>
+
+#include "Consts.h"
+#include "DiameterCodes.h"
+#include "LocalEnforcer.h"
+#include "includes/MagmaService.h"
+#include "Matchers.h"
+#include "ProtobufCreators.h"
+#include "includes/ServiceRegistrySingleton.h"
+#include "SessiondMocks.h"
+#include "SessionStore.h"
+
+using grpc::ServerContext;
+using grpc::Status;
+using ::testing::InSequence;
+using ::testing::Test;
+
+namespace magma {
+
+Teids teids0;
+Teids teids1;
+Teids teids2;
+
+class LocalEnforcerTest : public ::testing::Test {
+ protected:
+  virtual void SetUp() {
+    reporter      = std::make_shared<MockSessionReporter>();
+    pipelined_mock  = std::make_shared<MockPipelined>();
+    rule_store    = std::make_shared<StaticRuleStore>();
+    session_store = std::make_shared<SessionStore>(
+        rule_store, std::make_shared<MeteringReporter>());
+    pipelined_client     = std::make_shared<MockPipelinedClient>();
+    spgw_client          = std::make_shared<MockSpgwServiceClient>();
+    aaa_client           = std::make_shared<MockAAAClient>();
+    events_reporter      = std::make_shared<MockEventsReporter>();
+    auto default_mconfig = get_default_mconfig();
+    local_enforcer       = std::make_unique<LocalEnforcer>(
+        reporter, rule_store, *session_store, pipelined_client, events_reporter,
+        spgw_client, aaa_client, 0, 0, default_mconfig);
+    evb = folly::EventBaseManager::get()->getEventBase();
+    local_enforcer->attachEventBase(evb);
+    session_map   = SessionMap{};
+    test_cfg_     = get_default_config("");
```

Comment:
Do we use all the variables defined here?
test_cfg_ vs default_cfg_1 / default_cfg_2 for example. Let's just use one of them